plugin.rb

```diff
@@ -69,48 +69,12 @@ def self.s3_cdn_urls
 end
 
 after_initialize do
```

Comment:
This PR doesn't make any functional difference in terms of 'plugin disable-ability'? How much of it was 'forced' by the linter? vs. just a general refactor? I don't think we should prevent people from writing simple 'single-file plugins' if they want to.

Reply:
I’m just updating things where rubocop complains. So here it’s related to the rule that says to define all the monkey patches in a dedicated module. While writing everything in the `plugin.rb` file is certainly doable, I don’t think that’s a good approach. When the plugin grows, it quickly becomes a mess. We should encourage cleaner patterns IMO.

Reply:
(Discussed this in chat)

The Rubocop rule is blocking the use of `class_eval`, but devs are still free to use `prepend` in a `plugin.rb` file. Moving to separate files can be beneficial to make things more readable, especially as a plugin grows.